lib/pages/timetable/view/timetable_page.dart

```diff
@@ -291,6 +291,30 @@ class _TimetablePageState extends State<TimetablePage> {
           )
         ],
       );
+    } else if (!user.sources.contains('students')) {
+      return AppDialog(
+        title: S.current.warningNoEvents,
+        content: [
+          Text(
+            S.current.warningNoStudentSource,
```

Comment:
Nit: For simple things that fit on one line, don't add the trailing comma.

Reply:
got rid of that dirty comma